[2025-09-05T23:13:01.698+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-05T23:13:01.723+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T23:12:00+00:00 [queued]>
[2025-09-05T23:13:01.733+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T23:12:00+00:00 [queued]>
[2025-09-05T23:13:01.733+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-05T23:13:01.748+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-05 23:12:00+00:00
[2025-09-05T23:13:01.756+0000] {standard_task_runner.py:63} INFO - Started process 81955 to run task
[2025-09-05T23:13:01.760+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-05T23:12:00+00:00', '--job-id', '122285', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmp4o9pjgos']
[2025-09-05T23:13:01.763+0000] {standard_task_runner.py:91} INFO - Job 122285: Subtask save_foreign_1
[2025-09-05T23:13:01.820+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T23:12:00+00:00 [running]> on host d4d2fe648a81
[2025-09-05T23:13:01.917+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T23:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-05T23:12:00+00:00'
[2025-09-05T23:13:01.919+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-05T23:13:01.939+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T23:13:02.378+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-05T23:13:02.379+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T23:13:02.709+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-05T23:13:02.711+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T23:13:03.099+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-05T23:13:03.101+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T23:13:03.477+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-05T23:13:03.478+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-05T23:13:03.479+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-05T23:13:03.506+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-05T23:12:00+00:00, execution_date=20250905T231200, start_date=20250905T231301, end_date=20250905T231303
[2025-09-05T23:13:03.578+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-05T23:13:03.619+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-05T23:13:03.620+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
